test/test_prototype_transforms_functional.py

```diff
@@ -61,12 +61,7 @@ def make_info_args_kwargs_params(info, *, args_kwargs_fn, test_id=None):
     ]
 
 
-def make_info_args_kwargs_parametrization(infos, *, args_kwargs_fn, condition=None):
-    if condition is None:
```

Comment:
There is no need to have a condition that filters infos here, if we can simply filter them before passing them here.